ui/django-rq
# Added rqcronjobs management command to easily manage cron jobs from settings file

Idea here:
https://github.com/ui/django-rq/issues/159#issuecomment-216763329

* created `rqcronjobs` management command
* added some tests
* added some documentation

To verify:
* does it make sense to use lazy for `RQ_CRONJOBS`
* are all variable names/function names sane?
* should we remove onlyt cronjobs while iterating over scheduler or it's ok to just remove eveything? how to check that?
